Disabling istio sidecars injection (#14995)
kyma-project/kyma: docs/04-operation-guides/troubleshooting/service-mesh/smsh-04-istio-sidecar-version.md

## Cause

> By default, Kyma has sidecar injection disabled - a sidecar is not injected to any Pod in a cluster. For more information, read the document about [disabling Istio sidecar proxy injection](../../operations/smsh-01-istio-enable-sidecar-injection.md).

Reviewer: ```suggestion
By default, Kyma has sidecar injection disabled - there is no automatic sidecar injection into any Pod in a cluster. For more information, read the document about [enabling Istio sidecar proxy injection](../../operations/smsh-01-istio-enable-sidecar-injection.md).
```
